PHP image build fails

**Describe the bug**
The PHP image fails to build with an error `configure: error: jpeglib.h not found.`

**To Reproduce**
Steps to reproduce the behavior:
1. Execute `make build`.

--- a/php/Dockerfile
+++ b/php/Dockerfile
@@ -3,7 +3,7 @@ LABEL maintainer="Alexandre Jardin <info@ajardin.fr>"
 
 # Install Symfony requirements
 RUN \
-    apk add --no-cache --virtual .persistent-deps \
+    apk add --no-cache \
         freetype-dev \
         git \
         icu-libs \

